experimental/addons/harbor/README.md

```diff
@@ -0,0 +1,94 @@
+# Harbor
+
+[Harbor](https://goharbor.io/) is an open source registry that secures artifacts with policies and role-based access control, ensures images are scanned and free from vulnerabilities, and signs images as trusted.
+
+This addon install harbor by helm chart. 
+
+:::caution
+This is just a experimental addon that it didn't configure persistent storage for harbor, please make sure you're just using it for demo.
+:::
```

Comment:
This syntax is not supported by vanilla markdown. Please just use ">"

Reply:
We should support it in velaux